site/content/installation/installing-ngf/helm.md

```diff
@@ -41,14 +41,14 @@ To install the latest stable release of NGINX Gateway Fabric in the **nginx-gate
 
 - For NGINX Plus:
 
-  {{< note >}}Replace `private-registry.nginx.com` with the proper registry for your NGINX Plus image, and if applicable, replace `nginx-plus-registry-secret` with your Secret name containing the registry credentials. {{< /note >}}
+  {{< note >}}Replace `private-registry.nginx.com` with the proper registry for your NGINX Plus image, and if applicable, replace `nginx-plus-registry-secret` with your Secret name containing the registry credentials.{{< /note >}}
+
+  {{< note >}}Ensure that you [Enable Usage Reporting]({{< relref "installation/usage-reporting.md" >}}) when installing.{{< /note >}}
```

Comment:
There's other callouts that could make more sense for this, such as caution, important or warning.

If it has as much gravitas as the note on replacing registry, they can be combined into a singular paragraph.